sycl/source/detail/graph_impl.hpp

```diff
@@ -692,6 +714,9 @@ class exec_graph_impl {
     return false;
   }
 
+  /// @return true is emulated backend has been forced
+  bool isEmulationModeForced() const { return MGraphImpl->getEmulationModeForced(); }
```

Comment:
This line looks like it might be longer than 80 characters, is the file clang-format correct?